client/src/pages/UserAdmin.js

```diff
@@ -0,0 +1,318 @@
+            setUserManagedProjects([...userManagedProjects, projectValue]);
+            setProjectValue([]);
+        } else {
+            setProjectValue([]);
+        }
+    };
+
+    // Handle cancel form and return to search
+    const handleProjectFormCancel = () => {
+        setUserLoaded(false);
+        setUserToEdit({});
+        setSearchTerm("");
+        setUserManagedProjects([]);
+    };
+    
+    // Remove projects from db
+    const handleRemoveProject = (projectToRemove) => {
+        if (userManagedProjects.length > 0) {
+            setUserManagedProjects(userManagedProjects.filter(p => (p !== projectToRemove)));
+        } 
+    };
+
+    // This initially populates userManagedProjects upon selection of a user
+    const handleAddProject = (proj) => {
+        setUserManagedProjects(proj);
+    }
+
+    // Swaps the buttons and displayed panels for the search results, by email or by name
+    const buttonSwap = () => {
+        searchResultType === "email" 
+        ? setSearchResultType('name') 
+        : setSearchResultType('email');
+    };
+
+    // If there is a selected user, show the edit form; else show search form
+    if (Object.keys(userToEdit).length === 0) {
+        return (
+            <div className="container--usermanagement">
+                <div>
+                    <h3>User Management</h3>
+                    <input
+                        type="text"
+                        placeholder="Search by name and email..."
+                        value={searchTerm}
+                        onChange={handleChange}
+                    />
+                    <div className="tab-buttons">
+                        <div><button 
+                            className={searchResultType === "name" ? "select-button selected" : "select-button"} 
+                            onClick={buttonSwap} disabled={searchResultType === "name"}>Results by Name
+                        </button></div>
+                        <div><button 
+                            className={searchResultType === "email" ? "select-button selected" : "select-button"} 
+                            onClick={buttonSwap} disabled={searchResultType === "email"}>Results by Email
+                        </button></div>
+                    </div>
+                    <div>
+                        <div>
+                            {<ul className={searchResultType === "email" ? "search-results hide-results" : "search-results"}>    
+                                {nameResults.map((result,index) => {
+                                return (
+                                    <li key={index}>{result}</li>
+                                )})}
+                            </ul>}
+                        </div> 
+
+                        <div>
+                            {<ul className={searchResultType === "name" ? "search-results hide-results" : "search-results"}>
```

Comment:
As we discussed, I think making the searchResultType === "name" instead of "email" on line 193 and flipping where the hide-results class is added makes more sense in terms of code readability. Also on line 202, do the same witch changing "name" to "email" and flipping where hide-results class is added.

Reply:
code has been flipped.